app/_locales/en/messages.json

```diff
@@ -3182,6 +3182,9 @@
   "onlyConnectTrust": {
     "message": "Only connect with sites you trust."
   },
+  "openCustodianApp": {
+    "message": "Open $1 app"
```

Comment:
This needs a description key to indicate what $1 is.

Reply:
You're 100% right! Added it 💪🏼